solr/core/src/java/org/apache/solr/cloud/api/collections/OverseerRoleCmd.java

```diff
@@ -64,6 +64,10 @@ public void call(ClusterState state, ZkNodeProps message, NamedList<Object> resu
     SolrZkClient zkClient = zkStateReader.getZkClient();
     Map<String, List<String>> roles = null;
     String node = message.getStr("node");
+    if ("false".equals(message.getStr("persist"))) {// no need to persist to roles.json
```

Comment:
Is this a new feature? I don't see doc changes on it.

Reply:
This isn't a new feature, just a parameter added to the ADDROLE.